File: mi-carpeta/deber_5_if3.ts
// Saber si ya estas hecho papel higenico (estas ebrio)
console.log("Saber si ya estas hecho papel higenico")
const nuvielasString = prompt("Ingrese el numero de cervezas que hay bebido: ");
const nuvielas = Number(nuvielasString);
if(nuvielas>=1 && nuvielas<=3){
    console.log("Toadavia jalas ");
} else if(nuvielas>3 && nuvielas<=5){
    const respuesta3String = prompt(" (Si 1/ No 0)|| Quiere llamar a su ex?¿ : ");
    const respuesta3 = Number(respuesta3String);
    if(respuesta3 != 1 && respuesta3 != 0){
        console.log("Revise su respuesta!!")
    } else if(respuesta3 ==1){
        console.log("Que pendel@ anda a ver mejor master chef ")
    } else {
        console.log("Muy bien")
    }
    const respuesta2String = prompt("(Si 1/ No 0)|| Puede hablar : ");
    const respuesta2 = Number(respuesta2String);
    if(respuesta2 != 1 && respuesta2 != 0){
        console.log("Revise su respuesta!!")
    } else if(respuesta2 ==1){
        console.log("Una cerveza más ")
    } else {
        console.log("Dile a alguien que te de pidiendo un taxi/uber y andate a dormir")
    }
    const respuestaString = prompt("(Si 1/ No 0)|| Puede caminar : ");
    const respuesta = Number(respuestaString);
    if(respuesta != 1 && respuesta != 0){
        console.log("Revise su respuesta!!")
    } else if(respuesta ==1){
        console.log("Una cerveza más ")
    } else {
        console.log("Dile a alguien que te de pidiendo un taxi/uber y andate a dormir");
    }
  
} else if(nuvielas >5){
    console.log("Mejor pide taxi/uber y andate a dormir ")
}

File: mi-carpeta/deber_5_if3.go
// Saber si ya estas hecho papel higenico (estas ebrio)
package main

import (
	"fmt"
)

func main() {
	fmt.Println("Saber si ya estas hecho papel higenico")
	fmt.Println("Ingrese el numero de cervezas que hay bebido: ")
	var nuvi, resp int
	fmt.Scanln(&nuvi)
	if nuvi >= 1 && nuvi <= 3 {
		fmt.Println("Todavia jalas !!")
	} else if nuvi > 3 && nuvi <= 5 {
		fmt.Println("(Si 1/ No 0) / Quiere llamar a su ex ")
		fmt.Scanln(&resp)
		if resp != 1 && resp != 0 {
			fmt.Println("Revise su respuesta !!")
		} else if resp == 1 {
			fmt.Println("Que pendej@ anda a ver mejor master chef")
		} else {
			fmt.Println("Muy bien")
		}
		fmt.Println("(Si 1/ No 0) / Puede hablar ")
		fmt.Scanln(&resp)
		if resp != 1 && resp != 0 {
			fmt.Println("Revise su respuesta !!")
		} else if resp == 1 {
			fmt.Println("Una cerveza más")
		} else {
			fmt.Println("Dile a alguien que te de pidiendo un taxi/uber y andate a dormir")
		}
		fmt.Println("(Si 1/ No 0)  Puede caminar ")
		fmt.Scanln(&resp)
		if resp != 1 && resp != 0 {
			fmt.Println("Revise su respuesta !!")
		} else if resp == 1 {
			fmt.Println("Una cerveza más")
		} else {
			fmt.Println("Dile a alguien que te de pidiendo un taxi/uber y andate a dormir")
		}

	} else if nuvi > 5 {
		fmt.Println("Mejor pide taxi/uber y andate a dormir  ")
	}

}
